enarx-keep-sgx-shim/src/start.S

```diff
@@ -163,6 +198,9 @@ syscall:
 
     xor     %rcx,                   %rcx            # Clear %rcx
     zerof   %rcx                                    # Clear CPU flags
+    movq    $~0,                    %rax            # Set mask for xsaveopt in rax
+    movq    $~0,                    %rdx            # Set mask for xsaveopt in rdx
```

Comment:
You should probably only enable the supported bits from https://github.com/enarx/enarx/blob/master/sgx-types/src/attr.rs#L37 so that the size is not exceeded.